l3kernel/l3token.dtx

```diff
@@ -2245,6 +2282,46 @@
 %    \end{macrocode}
 % \end{macro}
 %
+% \begin{macro}[pTF]{\token_if_control_word:N}
+% \begin{macro}{\@@_if_control_word_aux:w}
+%   Check if token consists of the escape character followed by
+%   one or more letters.
+%   Hence we do not check catcode, we have to check if it is a
+%   character whose catcode is parameter, or much easier, we just test
+%   if it is a character. And |#1| could be a control symbol let to
```

Comment:
Cannot follow these sentences! 
Are they needed? 
But I see that the code is "pretty hairy", so maybe it does need an explanation! 
But not this one:-).